jsapp/js/account/stripe.api.ts

```diff
@@ -104,6 +104,9 @@ export async function postCustomerPortal(organizationId: string) {
 export async function getSubscriptionInterval() {
```

Comment:
It would be nice to add a short `/**` comment to all these functions here, so it's at least hinted what they do. I can guess some, but for example `postCustomerPortal` is completely puzzling